assets/dev/scss/frontend/_container.scss

```diff
@@ -33,27 +37,44 @@
 	--overlay-transition: 0.3s;
 
 	position: var( --position );
-	display: var( --display );
-	text-align: var( --text-align );
-	flex-direction: var( --flex-direction );
-	flex: var( --flex-grow ) var( --flex-shrink ) var( --flex-basis );
+	flex-direction: var( --container-flex-direction, var( --flex-direction ) );
+	flex: var( --container-flex-grow, var( --flex-grow ) ) var( --flex-shrink ) var( --flex-basis );
 	width: var( --width );
 	// Set `min-width` to fix nested Containers shrink bug (ED-4964).
 	// Ref: https://stackoverflow.com/questions/43934648/how-to-make-flexbox-items-shrink-correctly-when-in-a-nested-container
 	min-width: 0;
-	min-height: var( --min-height );
 	height: var( --height );
 	border-radius: var( --border-radius );
 	margin: var( --margin-top ) var( --margin-right ) var( --margin-bottom ) var( --margin-left );
-	// Custom content-width hack.
-	padding-left: Max( var( --padding-left ), calc( ( 100% - var( --content-width ) ) / 2 ) );
-	padding-right: Max( var( --padding-right ), calc( ( 100% - var( --content-width ) ) / 2 ) );
+	padding-left: var( --padding-left );
+	padding-right: var( --padding-right );
 	padding-top: var( --padding-top );
 	padding-bottom: var( --padding-bottom );
 	z-index: var( --z-index );
 	overflow: var( --overflow );
 	transition: var( --transition );
 
+	&, & > .e-container__inner {
+		display: var( --display );
+		min-height: var( --min-height );
+		text-align: var( --text-align );
+	}
+
+	&__inner {
+		flex-direction: var( --flex-direction );
+		flex-grow: var( --flex-grow );
+		flex-wrap: var( --flex-wrap );
+		justify-content: var( --justify-content );
+		align-items: var( --align-items );
+		align-content: var( --align-content );
+		gap: var( --gap );
+		align-self: var( --align-self );
+		width: 100%;
+		max-width: var( --content-width );
+		margin-inline: auto;
+		height: 100%;
```

Comment:
@StyleShit 

For direction: applied to inner container only (parent container has 'flex column' always)
As for other flex styling options: I fear this will have to be tested with a trial and error method, unless you have a better suggestion. What are your thoughts about this? We possibly could discuss this tomorrow in a video call.

Reply:
@StyleShit 

I have made a change in my approach. 
I have decided to use a prefix class. 

In this way I can organise the SCSS better. 

I know a prefix class isn't the preferred method, but I believe in this case it is a neither solution.